b2sdk/transfer/outbound/copy_manager.py

```diff
@@ -28,12 +28,12 @@ class CopyManager(LazyThreadPoolMixin):
 
     MAX_LARGE_FILE_SIZE = 10 * 1000 * 1000 * 1000 * 1000  # 10 TB
 
-    def __init__(self, services, max_copy_workers=10):
+    def __init__(self, services, max_workers: Optional[int] = None):
         """
         :param b2sdk.v2.Services services:
-        :param int max_copy_workers: maximum number of copy threads
+        :param int max_workers: maximum number of copy threads
         """
-        super().__init__(max_workers=max_copy_workers)
+        super().__init__(max_workers=max_workers)
         self.services = services
```

Comment:
this should be factored out to a parent class or something

Reply:
You mean `self.services`? Sure, why not.